.github/workflows/validate-version-bump.yaml

```diff
@@ -0,0 +1,23 @@
+name: Validate Chart Version Bump
+
+on:
+  pull_request:
+    paths:
+      - 'charts/opentelemetry-collector/Chart.yaml'
+  workflow_dispatch:
+
+jobs:
+  validate-version-bump:
+    permissions:
+      contents: read
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Validate chart version bump
+        run: |
+          bash charts/opentelemetry-collector/validate-chart-version-bump.sh
```

Comment:
If you `chmod +x` the script and commit it (git can store permissions like this), you can just call it with `./charts/opentelemetry-collector/validate-chart-version-bump.sh` because the file has a bang already.